csrankings-c.csv

```diff
@@ -470,6 +470,7 @@ Cheng Soon Ong,Australian National University,https://cs.anu.edu.au/people/cheng
 Cheng Tan 0005,Northeastern University,https://naizhengtan.github.io,OTzvsa4AAAAJ
 Cheng Yang 0002,BUPT,http://nlp.csai.tsinghua.edu.cn/~yangcheng,OlLjVUcAAAAJ
 Cheng Zhang 0011,Cornell University,https://infosci.cornell.edu/content/zhang-0,NOSCHOLARPAGE
+Cheng Zhuo,Zhejiang University,https://scholar.google.com/citations?user=-XSbEFAAAAAJ&hl=zh-CN&oi=ao,-XSbEFAAAAAJ

```

Comment:
Google Scholar pages are not appropriate substitutes for home pages. Please re-open with a departmental page which will also let me establish their eligibility for inclusion.